pkg/pillar/cmd/upgradeconverter/applydefaultconfigitem.go

```diff
@@ -4,16 +4,24 @@
 package upgradeconverter
 
 import (
+	"bufio"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
+	"strings"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	fileutils "github.com/lf-edge/eve/pkg/pillar/utils/file"
 )
 
+const (
+	baseAuthorizedKeysFile = types.IdentityDirname + "/authorized_keys"
+	globalConfig           = types.IdentityDirname + "/GlobalConfig/global.json"
```

Comment:
globalConfig is a bit too generic name. How about calling it importGlobalConfigFile and the other importAuthrotizedKeysFile?

Reply:
Sorry. Last commit was not a complete one. I was trying to sync my sandbox between linux & mac machines. Diff gave issues and I resorted to commit  and then pull from different machine. Forgot that I already have a PR open for it.